tron/bin/check_tron_jobs.py

```diff
@@ -115,23 +126,20 @@ def compute_check_result_for_job_runs(client, job, job_content):
         prefix = "UNKNOWN: The job is in a state that check_tron_jobs doesn't understand"
         status = 3
 
+    precious_runs_note = ''
+    if _are_runs_precious(job['monitoring']):
+        precious_runs_note = f"Note: Job {job_content['name']} is precious. Owners will be alerted if a day's, instead of the overall job's, latest run failed.\n"
+
     kwargs["output"] = (
-        "{}\n"
-        "{}'s last relevant run (run {}) {}.\n\n"
-        "Here is the last action:"
-        "{}\n\n"
+        f"{prefix}\n"
+        f"{job['name']}'s last relevant run (run {relevant_job_run['id']}) {relevant_job_run['state']}.\n"
```

Comment:
I would like to have the date here too if it is precious.

`name is the latest run for $date ($dom) {state}.`